spi/src/main/java/cf/dmms/app/spi/Book.java

```diff
@@ -1,7 +1,6 @@
-package cf.dmms.app.core.book;
+package cf.dmms.app.spi;
```

Comment:
Why did you move Book from core to spi?

Reply:
I thought that only user was supposed to be moved, and then core would include spi

Reply:
core does include spi, but having entities all around is not a good idea in my opinion. Also, if user is in SPI why book is not? Some kind of logic problem here I suppose